src/dsp_tools/cli.py

```diff
@@ -517,9 +517,10 @@ def _call_requested_action(args: argparse.Namespace) -> bool:
             path_to_output_file=args.properties_section,
         )
     elif args.action == "id2iri":
-        success = id_to_iri(
+        success = id2iri(
             xml_file=args.xmlfile,
             json_file=args.mapping,
+            remove_resource_if_id_in_mapping=args.remove_resources,
```

Comment:
This one hurts. In the original PR that introduced this feature, I didn't follow Test Driven Development, so I merged a non-functional feature into main, just because I forgot this single line of code.
Thankfully, I realized today that I need a test, and found this bug 🙈